Source: https://gitlab.freedesktop.org/xkeyboard-config/xkeyboard-config/commit/af6c0f29dbee1cc7b280eed891592797c2e31220
Upstream: yes, releases are made every 4 months
Reason: backticks broke it

diff --git symbols/pt symbols/pt
index 73a1b7b7b6dec730704e3efadf03790a80b4b14b..3ec09812c1b859db134efc848995f01ffa775eee 100644
--- symbols/pt
+++ symbols/pt
@@ -221,35 +221,35 @@ xkb_symbols "colemak" {
 
     name[Group1]="Portuguese (Colemak)";
 
-    key `<AD01>`  { [         q,          Q,           at,   Greek_OMEGA ] };
-    key `<AD02>`  { [         w,          W,      lstroke,       Lstroke ] };
-    key `<AD03>`  { [         f,          F,      dstroke,   ordfeminine ] };
-    key `<AD04>`  { [         p,          P,        thorn,         THORN ] };
-    key `<AD05>`  { [         g,          G,          eng,           ENG ] };
-    key `<AD06>`  { [         j,          J,    dead_hook,     dead_horn ] };
-    key `<AD07>`  { [         l,          L,      lstroke,       Lstroke ] };
-    key `<AD08>`  { [         u,          U,    downarrow,       uparrow ] };
-    key `<AD09>`  { [         y,          Y,    leftarrow,           yen ] };
-    key `<AD10>`  { [  ccedilla,   Ccedilla,   dead_acute, dead_doubleacute ] };
-
-    key `<AC01>`  { [         a,          A,           ae,           AE ] };
-    key `<AC02>`  { [         r,          R,    paragraph,   registered ] };
-    key `<AC03>`  { [         s,          S,       ssharp,      section ] };
-    key `<AC04>`  { [         t,          T,       tslash,       Tslash ] };
-    key `<AC05>`  { [         d,          D,          eth,          ETH ] };
-    key `<AC06>`  { [         h,          H,      hstroke,      Hstroke ] };
-    key `<AC07>`  { [         n,          N,            n,            N ] };
-    key `<AC08>`  { [         e,          E,     EuroSign,         cent ] };
-    key `<AC09>`  { [         i,          I,   rightarrow,     idotless ] };
-    key `<AC10>`  { [         o,          O,       oslash,     Ooblique ] };
-
-    key `<AB01>`  { [         z,          Z, guillemotleft,        less ] };
-    key `<AB02>`  { [         x,          X, guillemotright,    greater ] };
-    key `<AB03>`  { [         c,          C,         cent,    copyright ] };
-    key `<AB04>`  { [         v,          V, leftdoublequotemark, leftsinglequotemark ]   };
-    key `<AB05>`  { [         b,          B, rightdoublequotemark, rightsinglequotemark ] };
-    key `<AB08>`  { [         k,          K,          kra,    ampersand ] };
-    key `<AB07>`  { [         m,          M,           mu,    masculine ] };
-
-    key `<CAPS>` { [    BackSpace,    BackSpace,       BackSpace,        BackSpace ] };
+    key <AD01>  { [         q,          Q,           at,   Greek_OMEGA ] };
+    key <AD02>  { [         w,          W,      lstroke,       Lstroke ] };
+    key <AD03>  { [         f,          F,      dstroke,   ordfeminine ] };
+    key <AD04>  { [         p,          P,        thorn,         THORN ] };
+    key <AD05>  { [         g,          G,          eng,           ENG ] };
+    key <AD06>  { [         j,          J,    dead_hook,     dead_horn ] };
+    key <AD07>  { [         l,          L,      lstroke,       Lstroke ] };
+    key <AD08>  { [         u,          U,    downarrow,       uparrow ] };
+    key <AD09>  { [         y,          Y,    leftarrow,           yen ] };
+    key <AD10>  { [  ccedilla,   Ccedilla,   dead_acute, dead_doubleacute ] };
+
+    key <AC01>  { [         a,          A,           ae,           AE ] };
+    key <AC02>  { [         r,          R,    paragraph,   registered ] };
+    key <AC03>  { [         s,          S,       ssharp,      section ] };
+    key <AC04>  { [         t,          T,       tslash,       Tslash ] };
+    key <AC05>  { [         d,          D,          eth,          ETH ] };
+    key <AC06>  { [         h,          H,      hstroke,      Hstroke ] };
+    key <AC07>  { [         n,          N,            n,            N ] };
+    key <AC08>  { [         e,          E,     EuroSign,         cent ] };
+    key <AC09>  { [         i,          I,   rightarrow,     idotless ] };
+    key <AC10>  { [         o,          O,       oslash,     Ooblique ] };
+
+    key <AB01>  { [         z,          Z, guillemotleft,        less ] };
+    key <AB02>  { [         x,          X, guillemotright,    greater ] };
+    key <AB03>  { [         c,          C,         cent,    copyright ] };
+    key <AB04>  { [         v,          V, leftdoublequotemark, leftsinglequotemark ]   };
+    key <AB05>  { [         b,          B, rightdoublequotemark, rightsinglequotemark ] };
+    key <AB08>  { [         k,          K,          kra,    ampersand ] };
+    key <AB07>  { [         m,          M,           mu,    masculine ] };
+
+    key <CAPS> { [    BackSpace,    BackSpace,       BackSpace,        BackSpace ] };
 };
